orttraining/orttraining/training_ops/cuda/nn/dropout_impl.cu

```diff
@@ -65,5 +73,103 @@ SPECIALIZED_DROPOUT_GRAD_IMPL(float)
 SPECIALIZED_DROPOUT_GRAD_IMPL(double)
 SPECIALIZED_DROPOUT_GRAD_IMPL(half)
 
+constexpr int UNROLL = 4;
```

Comment:
>UNROLL [](start = 14, length = 6)

how about something like `k_unroll_count`?

Reply:
I think it's fine. :) 

---
In reply to: [441088010](https://github.com/microsoft/onnxruntime/pull/4167#discussion_r441088010) [](ancestors = 441088010)